.travis.yml

```diff
@@ -3,9 +3,9 @@ go:
   - 1.7
 install: make install-ci
 env:
- # Set higher timeouts for Travis
- - TEST_TIMEOUT_SCALE=20
- - PACKAGE=github.com/m3db/m3collector
+  global:
+    - TEST_TIMEOUT_SCALE=20 # Set higher timeouts for Travis
```

Comment:
I think what we want is to have both env vars set in the same line therefore triggering a single build, i.e., `- TEST_TIMEOUT_SCALE=20 PACKAGE=github.com/m3db/m3aggregator`

Reply:
That's what I did initially actually, but I think global is appropriate for these variables because they should apply to every run. Even though we only have one run at the moment if we ever add environment variables that should be set on specific runs we still want these to be set on those runs as well.